guides/common/modules/proc_passing-arguments-to-shellhook-script-using-curl.adoc

```diff
@@ -0,0 +1,22 @@
+[id="passing-arguments-to-shellhook-script-using-curl_{context}"]
+= Passing Arguments to Shellhook Script Using Curl
+
+Use this procedure to pass arguments to a shellhook script using curl.
+
+.Procedure
+* When executing a shellhook script using curl, create HTTP headers in the following format:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+"X-Shellhook-Arg-1: _VALUE_"
+"X-Shellhook-Arg-2: _VALUE_"
+----
+
+.Example
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+# curl -sX POST -H 'Content-Type: text/plain' \
+-H "X-Shellhook-Arg-1: Version 1.0" \
+-H "X-Shellhook-Arg-2: My Content View" \
+--data "" https://localhost:9090/shellhook/_my_script_
```

Comment:
```suggestion
--data "" https://localhost:9090/shellhook/_My_Script_
```

Reply:
Actually, I realized it was not supposed to be italicized since it is an example. Updated name of the script :+1: